tests/SixLabors.Fonts.Tests/SixLabors.Fonts.Tests.csproj

```diff
@@ -8,6 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="FluentAssertions" Version="6.10.0" />
```

Comment:
I don’t use this library and don’t want to introduce it as I’m not a fan of the style